array.c

```diff
@@ -935,21 +935,26 @@ rb_check_to_array(VALUE ary)
 
 /*
  *  call-seq:
- *     Array.try_convert(obj) -> array or nil
+ *    Array.try_convert(obj) -> new_array or nil
  *
- *  Tries to convert +obj+ into an array, using the +to_ary+ method.  Returns
- *  the converted array or +nil+ if +obj+ cannot be converted.
- *  This method can be used to check if an argument is an array.
+ *  Tries to convert <tt>obj</tt> to an \Array.
```

Comment:
I think you should stick with `+obj+` as it is a parameter, and `+` is rdoc equivalent to Markdown `` ` ``, makes it easier to read in source-form, and lets RDoc more-easily audit documentation completeness with `rdoc -C2`

Reply:
Done, throughout.